rdkit/Chem/MolStandardize/charge.py

```diff
@@ -245,16 +243,10 @@ def reionize(self, mol):
         return mol
 
     def _strongest_protonated(self, mol):
-        for position, pair in enumerate(self.acid_base_pairs):
-            for occurrence in mol.GetSubstructMatches(pair.acid):
-                return position, occurrence
-        return None, None
+        return 0, mol.GetSubstructMatches(self.acid_base_pairs[0].acid)
```

Comment:
This looks very different!

Reply:
The old code iterate only 1 time and only return at the beginning. If you think it may indicate some incomplete function, I will revert it.